### PR TITLE
[Settings] Following feed prefs revamp

### PR DESCRIPTION
**Pull request by mozzius**
# Stacked on #5772 

Simulator Screenshot - iPhone 16 - 2024-10-16 at 16 05 57

### CODE REVIEW

File: src/screens/Settings/FollowingFeedPreferences.tsx [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+              <SettingsList.ItemText>
+                <Trans>Show reposts</Trans>
+              </SettingsList.ItemText>
+              <Toggle.Platform />
+            </SettingsList.Item>
+          </Toggle.Item>
+          <Toggle.Item
+            type="checkbox"
+            name="show-quotes"
+            label={_(msg`Show quotes`)}
```
Comment (surfdude29) [inline src/screens/Settings/FollowingFeedPreferences.tsx:97]:
```suggestion
            label={_(msg`Show quote posts`)}
```
Small tweak suggestion: what about aligning the label text with the displayed text?

File: src/screens/Settings/FollowingFeedPreferences.tsx [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+          <SettingsList.Group>
+            <SettingsList.ItemIcon icon={BeakerIcon} />
+            <SettingsList.ItemText>
+              <Trans>Show posts from my feeds (experimental)</Trans>
+            </SettingsList.ItemText>
+            <Toggle.Item
+              type="checkbox"
+              name="merge-feed"
+              label={_(
+                msg`Show samples of your saved feeds in your Following feederge feed`,
```
Comment (surfdude29) [inline src/screens/Settings/FollowingFeedPreferences.tsx:122]:
```suggestion
                msg`Show samples of your saved feeds in your Following feed`,
```

File: src/screens/Settings/FollowingFeedPreferences.tsx [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+  )
+
+  return (
+    <Layout.Screen testID="followingFeedPreferencesScreen">
+      <Layout.Header title={_(msg`Following Feed Preferences`)} />
+      <Layout.Content>
+        <SettingsList.Container>
+          <SettingsList.Item>
+            <Admonition type="tip" style={[a.flex_1]}>
+              These settings only apply to the Following feed.
```
Comment (surfdude29) [inline src/screens/Settings/FollowingFeedPreferences.tsx:55]:
```suggestion
              <Trans>These settings only apply to the Following feed.</Trans>
```

**Comment by haileyok** [comment]
I wonder if we want to just call this `Experimental` for a header then have individual toggles below?

Screenshot 2024-10-31 at 12 07 40 PM

**Review by haileyok — APPROVED** [review]

**Pull request merged**